presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Partition.java

```diff
@@ -56,7 +57,8 @@ public Partition(
             @JsonProperty("partitionVersion") Optional<Long> partitionVersion,
             @JsonProperty("eligibleToIgnore") boolean eligibleToIgnore,
             @JsonProperty("sealedPartition") boolean sealedPartition,
-            @JsonProperty("createTime") int createTime)
+            @JsonProperty("createTime") int createTime,
+            @JsonProperty("lastDataCommitTime") int lastDataCommitTime)
```

Comment:
what's the difference between createTime and lastDataCommitTime?  Does it depend on whether the partition is immutable?

Reply:
> what's the difference between createTime and lastDataCommitTime? Does it depend on whether the partition is immutable?

Yeah, I think it depends on the immutability of the partition. (The details I need to refer to metastore, since they maintain the underlying logic).